### DISCUSSION TITLE
release 2.2.0

### DISCUSSION THREAD

**Discussion by Aloren** [aerospike/spring-data-aerospike#518] [Team Posts]
Hello @rbotzer
We would like to release a new version of spring-data-aerospike with major upgrade to 2.2.0 spring-data-parent.

I've created release draft: https://github.com/aerospike-community/spring-data-aerospike/releases/tag/untagged-29783ec9d42ebafdef24
and updated version in master: 
https://github.com/aerospike-community/spring-data-aerospike/commit/40c39ac18a69c700f7e2a62b26c6fb2e5846bfc5

**Comment by rbotzer** [comment]
@Aloren Brian Nichols pushed it to Maven Central. Should be available in a few hours.
